src/main/resources/assets/traverse/lang/zh_cn.json

```
@@ -3,20 +3,20 @@
 	"biome.traverse.autumnal_wooded_hills":"秋木山脉",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|
```suggestion
	"biome.traverse.autumnal_wooded_hills":"秋木丘陵",
```
同下。

---

秋森丘陵比较好。